extensions/test/integration/healthcheck/healthcheck_operation.go

```diff
@@ -84,7 +84,18 @@ func ExtensionHealthCheckWithManagedResource(ctx context.Context, timeout time.D
 		healthConditionType)
 }
 
-func testHealthCheckWithManagedResource(ctx context.Context, timeout time.Duration, f *framework.ShootFramework, extensionKind schema.GroupVersionKind, extensionName string, managedResourceName string, healthConditionType gardencorev1beta1.ConditionType) error {
+func ContainerRuntimeHealthCheckWithManagedResource(ctx context.Context, timeout time.Duration, f *framework.ShootFramework, extensionName string, managedResourceName string, healthConditionType gardencorev1beta1.ConditionType) error {
```

Comment:
Looks good, can you add doc strings for all the exported functions in this file?